epaint/src/textures.rs

```diff
@@ -150,7 +153,7 @@ impl TextureMeta {
 #[must_use = "The painter must take care of this"]
 pub struct TexturesDelta {
     /// New or changed textures. Apply before painting.
-    pub set: AHashMap<TextureId, ImageDelta>,
+    pub set: AHashMap<TextureId, Vec<ImageDelta>>,
```

Comment:
Maybe it would be even simpler to just make this a `Vec<(TextureId, ImageDelta)>`? I don't really see the point of having it be a hashmap anymore.

Reply:
Excellent suggestion. That avoids the double allocations. However, we would then always push to the end, and pop the front, like a FIFO, right? If so, maybe using a [`VecDequeue`]([VecDeque](https://doc.rust-lang.org/std/collections/struct.VecDeque.html#)) would be clever? On the other hand, we always consume the full `Vec` at once when we consume it, so using `into_iter()` will probably be even more efficient than the VecDequeue, then again. Does `into_iter()` on a normal vector cause any de-allocations?

Reply:
A `Vec` is the best fit since we always push to the back and consume the whole thing with either `std::mem::take` or `drain`